Resources/spine/Nya.atlas

```diff
@@ -0,0 +1,201 @@
+
+Nya.png
+size: 1024,1024
+format: RGBA8888
+filter: Linear,Linear
+repeat: none
+antepierna1
```

Comment:
@protito Would it be possible to translate the names of the body parts from Spanish to English?